build.js

```diff
@@ -33,18 +33,43 @@ run(browserify + ' .temp.js', function(error, stdout) {
   ].join('\n');
 
   // Use the online Google Closure Compiler service for minification
-  fs.writeFileSync('.temp.js', querystring.stringify({
+  var body = Buffer.from(querystring.stringify({
     compilation_level: 'SIMPLE_OPTIMIZATIONS',
     output_info: 'compiled_code',
     output_format: 'text',
     js_code: code
   }));
-  run('curl -d @.temp.js "https://closure-compiler.appspot.com/compile"', function(error, stdout) {
-    if (error) throw error;
-    var code = header + '\n' + stdout;
-    fs.unlinkSync('.temp.js');
-    fs.writeFileSync('browser-source-map-support.js', code);
-    fs.writeFileSync('amd-test/browser-source-map-support.js', code);
+  console.log('making request to google closure compiler');
+  var https = require('https');
```

Comment:
I considered stashing the response in `node_modules/.cache` somewhere and only making the request if the input file changed since the last one, but figured that might be overkill?

Reply:
Yeah, I think that we should replace this with a local build system anyways, so no worries for now ☺️